tools/experiments/experiments.js

```diff
@@ -451,14 +457,75 @@ if (getMode().localDev) {
  * Builds the expriments tbale.
  */
 function build() {
-  const table = document.getElementById('experiments-table');
+  const {host} = window.location;
+
+  const subdomain = document.getElementById('subdomain');
+  subdomain.textContent = host;
+
+  // #redirect contains UI that generates a subdomain experiments page link
+  // given a "google.com/amp/..." viewer URL.
+  const redirect = document.getElementById('redirect');
+  const input = redirect.querySelector('input');
+  const button = redirect.querySelector('button');
+  const anchor = redirect.querySelector('a');
+  button.addEventListener('click', function() {
+    const ampUrl = viewerToAmpUrl(input.value);
```

Comment:
I think we should switch to folks just entering their domain and going from there instead of mentioning the AMP viewer.

Reply:
How about removing the viewer mention but we detect and support both?

Copy/pasting a viewer URL avoids typing.

Reply:
That might suggest it would work in Safari (which it won't). I'd really go with simple here.

Reply:
Sorry, which part won't work in Safari? 

Viewer URL detection is just "is the domain google.com?". We can even add support for Bing viewer here, though their experiments page is [currently broken](https://www.bing-amp.com/experiments.html). :)

Anyways I removed it, but some mobile testing workflows will suffer.

Reply:
In Safari, localStorage in a top-window (cache URL) isn't visible when the same origin loads in a viewer.

Reply:
TIL 🤦‍♂